vm/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -3,75 +3,57 @@
-* Persistent queues are slower but reliable.
+When running on a single instance, persistent queues work by serializing and storing the contents on the disk. However, when running in cluster mode, the memory grid backs the persistent queues. Thereby, when a flow uses the VM Connector to publish content to a queue, Mule runtime engine determines whether to process the message in the same origin node or to send the message to the cluster to pick another node. The VM Connector facilitates the load distribution across a cluster.
+In either case, when using persistent queues the contents you send must be serializable. Even though you can enable Kryo serialization to get a wider range of serializable values, Kryo does have some limitations. Therefore, try to:
 
-When running on a single instance, persistent queues work by serializing and storing the contents on the disk.
+* Keep your values simple because complex structures might result in serialization errors or performance issues if they are not easy to serialize
+* Make sure complex Java object implements the `Serializable` interface and that they conform to the Java Bean contract
+* Use streams, JSON objects, maps, and so on as long as the associated values comply with the previous recommendations
 
-When running in cluster mode, persistent queues are backed by the memory grid. This means that when a flow uses the VM connector to publish content to a queue, the Mule runtime determines whether to process the message in the same origin node or to send it to the cluster for another node to pick up. Using the VM module makes it easy to distribute the load across a cluster.
+== Prerequisites
 
-Either way, transactions are always supported.
+To use this connector, you must be familiar with:
 
-Release Notes: xref:release-notes::connector/connector-vm.adoc[VM Connector Release Notes] +
-Exchange: https://www.mulesoft.com/exchange/org.mule.connectors/mule-vm-connector/[VM Connector]
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
 
-== POM File Information
+Before creating an app, you must have access to Anypoint Studio.
 
-[source,xml,linenums]
-----
-<dependency>
-  <groupId>org.mule.connectors</groupId>
-  <artifactId>mule-vm-connector</artifactId>
-  <version>RELEASE</version>
-  <classifier>mule-plugin</classifier>
-</dependency>
-----
+== Common Use Cases for the Connector
 
-Mule converts RELEASE to the latest version. To specify a version, view
-Anypoint Exchange and click *Dependency Snippets*.
+Use the VM Connector when you want to:
 
-== When to Use the VM Connector
+* Pass messages from one flow to another through a queuing mechanism, instead of using a `<flow-ref />` directly
+* Distribute work across a cluster
+* Communicate with different apps that are running in the same Mule domain
+* Perform simple queueing that does not justify the usage of a full JMS broker
```

Comment:
I wonder if the inclusion of this list does not remove the need for all the intro text about different types of queues and how they work...?

Reply:
```suggestion
* Pass messages from one flow to another through a queuing mechanism, instead of using `<flow-ref />` directly
* Distribute work across a cluster
* Communicate with different apps that are running in the same Mule domain
* Perform simple queueing that does not justify the use of a full JMS broker
```